chain/app/app.go

```diff
@@ -123,6 +122,11 @@ type bandApp struct {
 	paramsKeeper   params.Keeper
 	zoracleKeeper  zoracle.Keeper
 
+	// Decoder for unmarshaling []byte into sdk.Tx
+	txDecoder sdk.TxDecoder
+	// Deliver Context that is set during BeingBlock and unset during EndBlock; primarily for gas refund
+	deliverContext sdk.Context
```

Comment:
Make it to public or create getter function to get it

Reply:
That will be on a separate PR to make everything including all keepers accessible.